HdfsAccessor.go

```diff
@@ -82,21 +81,21 @@ func (this *hdfsAccessorImpl) ConnectMetadataClient() error {
 // Establishes connection to a name node in the context of some other operation
 func (this *hdfsAccessorImpl) ConnectToNameNode() (*hdfs.Client, error) {
 	// connecting to HDFS name node
-	nnAddr := this.NameNodeAddresses[this.CurrentNameNodeIdx]
-	client, err := this.connectToNameNodeImpl(nnAddr)
+	client, err := this.connectToNameNodeImpl()
```

Comment:
Can we have some comments here explaining the logic (because colinmarc's hdfs supports multiple name node)?

Reply:
has uploaded